src/api/operator/operatorApi.ts

```diff
@@ -133,12 +133,12 @@ export async function getTrades(
     orderId?: UID
   } & WithNetworkId,
 ): Promise<RawTrade[]> {
-  const { networkId, owner, orderId } = params
-  console.log(`[getTrades] Fetching trades on network ${networkId} with filters`, { owner, orderId })
+  const { networkId, owner, orderId: orderUid } = params
+  console.log(`[getTrades] Fetching trades on network ${networkId} with filters`, { owner, orderUid })
 
-  const tradesPromise = orderBookSDK.getTrades({ owner, orderId }, { chainId: networkId })
+  const tradesPromise = orderBookSDK.getTrades({ owner, orderUid }, { chainId: networkId })
   const tradesPromiseBarn = orderBookSDK
-    .getTrades({ owner, orderId }, { chainId: networkId, env: 'staging' })
+    .getTrades({ owner, orderUid }, { chainId: networkId, env: 'staging' })
```

Comment:
When did this parameter change?
It should be at least a minor version bump, no?

Reply:
Not really, you can just do a bump with no changes, mentioning the reason in the release description